sql/telemetry_derived/devtools_accessiblility_panel_usage_v1/query.sql

```diff
@@ -0,0 +1,133 @@
+WITH accessibility_panel_client_days AS (
+  SELECT
+    submission_date,
+    normalized_channel,
+    payload.processes.parent.scalars.devtools_accessibility_opened_count AS opened_count,
```

Comment:
I think I'm probably the only frequent user of the dashboard. It is still on the team's/product's radar since, historically, other devtools dashboards do not really include much of the telemetry from the accessibility panel.

Reply:
👍 let's stay consistent with the devtools panel then